grr/server/grr_response_server/gui/ui/lib/api_translation/primitive_test.ts

```diff
@@ -81,7 +81,7 @@ describe('decodeBase64', () => {
 
   it('throws an error on invalid input', () => {
     expect(() => decodeBase64('Inv@lid $tring'))
-        .toThrow(new Error('Invalid base64 encoding.'));
+        .toThrowError('Failed to execute \'atob\' on \'Window\': The string to be decoded is not correctly encoded.');
```

Comment:
This is a browser implementation detail that we likely do not depend upon: There is no downside for us, if the error message is not exactly the same string. If we test in a different browser or the browser changes this error message with a version update, the test breaks. Since you do not alter the thrown Error, it is fine here to only expect any Error to be thrown.